keycloak/keycloak-admin-ui
# Create attribute - required toggle in general settings

## Motivation
Added switch logic as per the old console. Setting the "Required" toggle to false should hide the "Required For" and "Required When" fields. Related Design https://marvelapp.com/prototype/5eed532/screen/82246809

## Verification Steps

1. Go to realm-settings, user profile tab (if not present, enable it in general settings)
2. Create a new attribute and scroll down to the "Required" toggle. Verify that when the toggle is set to false the "Required For" and "Required When" fields are hidden. Set the toggle to true, complete the fields and save the attribute. 

## Checklist:
